lib/evss/pciu/phone_number.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module EVSS
+  module PCIU
+    class PhoneNumber < BaseModel
```

Comment:
@jkassemi @kreek - You'll note that I created this new `PhoneNumber` model in the `lib/evss/pciu` directory, vs the `apps/model` directory.

I have seen examples of both.  For EVSS specifically, the similar models are in the `lib/evss/pciu_address/` directory (i.e. [domestic_address.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/pciu_address/domestic_address.rb)).

I will be adding a few more similar models in the upcoming stories for email and alternate phone.

Would you recommend keeping these models here, or moving them to the `apps/models` directory?

Reply:
I see the services in `lib` as implementations that could technically be isolated into their own gems as they grow (like https://github.com/department-of-veterans-affairs/ruby-bgs), and which contain models that represent objects with a 1<>1 to the partner service we integrate with. In that sense, I think it makes sense to keep them in the `lib/evss` directory, while maintaining models that our application natively implements (saved forms, subscriptions, beta registrations, user notifications, etc) in `app/models`.

Reply:
+1000000 on that direction - external integration services should be as self contained as possible and provide a clean interface to the rest of the application. One of the low-hanging modularizations I think we need to do for the platform work.

Reply:
I think this fits in well with how we currently have some models set up. For example, we have models for MVI in the `lib/mvi` directory that implement the data structure we parse out of the response, but we then have a facade model `app/models/mvi.rb` that implements caching, helper methods, and makes use of `vets-api` specific features we might need.